scripts/accept.ts

```diff
@@ -25,7 +25,8 @@ export const accept = async () => {
     const vsk = await makeVstorageKit({ fetch }, {
         rpcAddrs: [AGORIC_RPCS[ACTIVE_AGORIC_RPC_INDEX]], network: AGORIC_NETWORK
     });
-    const instance = vsk.agoricNames.instance.fastUsdc;
+    const agoricNames = await vsk.readPublished('agoricNames');
+    const instance = agoricNames.instance.fastUsdc;
```

Comment:
This doesn't look right to me; there's no data stored at [`agoricNames`](https://vstorage.agoric.net/?path=published.agoricNames&endpoint=https%3A%2F%2Fmain-a.rpc.agoric.net%3A443&height=undefined) and [`readPublished`](https://github.com/Agoric/agoric-sdk/blob/bd136ede475826a9465d3f431a4fdc1ff9ab1b7b/packages/client-utils/src/vstorage-kit.js#L128) doesn't seem to do anything w.r.t. children.
```suggestion
    const namedInstances = await vsk.readPublished('agoricNames.instance');
    const instance = namedInstances.find(
      ([label, _instance]) => label === 'fastUsdc',
    )?.[1];
```
and likewise in src/lib/agoric.ts.

Reply:
We might just want a helper, e.g.
```ts
const agoricNamesLookup = async (
  vsk: Awaited<ReturnType<typeof makeVstorageKit>>,
  collectionName:
    | 'brand'
    | 'installation'
    | 'instance'
    | 'issuer'
    | 'oracleBrand'
    | 'vbankAsset',
  itemName: string,
): Promise<RemotableObject<any> | undefined> => {
  const collection = (await vsk.readPublished(
    `agoricNames.${collectionName}`,
  )) as Array<[string, RemotableObject<any>]>;
  for (const [name, item] of collection) {
    if (name === itemName) return item;
  }
};
```
(and something of the sort could already exist)

Reply:
Could we not do 

```js
    const namedInstances = await vsk.readPublished('agoricNames.instance');
    const instances = Object.fromEntries(namedInstances);
    const instance = instance.fastUsdc;
```

Reply:
Didn't use the helper approach, and pushed the initial suggestion.

Reply:
@gibson042 PTAL

Reply:
Working now from our end